algolia/query_suggestions/client_config.go

```diff
@@ -0,0 +1,68 @@
+package query_suggestions
+
+import (
+	"fmt"
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/call"
+	"net/http"
+)
+
+type IndexConfig struct {
```

Comment:
```suggestion
type IndexConfiguration struct {
```
We encourage to use the full term names for models. 
It's still appropriate to use the `Config` in the methods names as they match the actual API endpoints paths.

Reply:
I see, I replaced it with `IndexConfiguration`.